datashuttle/utils_mod/utils.py

```diff
@@ -313,23 +258,55 @@ def process_names(
     return prefixed_names
 
 
-def update_ses_names_with_datetime(names: list):
+def update_names_with_datetime(names: list):
```

Comment:
A very easy mistake for the user to make is to type lowercase, e.g. `@date`. Consider either implicitly capitalising (being case-insensitive) or throwing a warning